app/index.html

```diff
@@ -84,60 +86,100 @@ <h2 class="subtitle">
-      <div class="vertical-space-lg"></div>
-      <a href="contact.html" class="button uppercase">Work with us</a>
-    </section>
-
     <!-- Services section -->
     <section id="services" class="inset-1220">
-      <h1 class="uppercase section-title">What we can do for you</h1>
-
-      <div class="service reverse">
-        <img src="./images/web.jpg" alt="Desk and computer with website on screen">
-        <div class="service-description right">
-          <h2>Web</h2>
-          <p>We create functional, elegant websites and web apps that increase customer engagement.</p>
-          <p>Our responsive designs look great on any platform, from desktop to mobile and tablet.</p>
+      <h1 class="uppercase section-title">Areas of Expertise</h1>
+
+      <div class="service">
+        <div class="service-description">
+          <img src="./images/notes.svg" alt="notes with paperclip" width="120px" />
+          <h2>Product Design Sprints</h2>
+          <p>Launching something before validating your ideas can be expensive and risky. Don’t wait until your product goes to
+          market. Start your project with a design sprint—a structured project kick-off that defines goals, aligns the team, and
+          ends with a user-tested prototype.</p>
         </div>
+        <aside>
+          <img src="./images/notes.svg" alt="notes with paperclip" width="79px" />
+          <ul>
+            <li>Interactive mockups</li>
+            <li>Rapid prototyping</li>
+            <li>Strategy</li>
+            <li>Design thinking</li>
+          </ul>
+        </aside>
       </div>
 
       <div class="service">
-        <img src="./images/mobile.jpg" alt="Mobile phone with paper mockups">
         <div class="service-description">
-          <h2>Mobile</h2>
-          <p>Our intuitive and beautiful mobile applications work across both iOS and Android platforms. </p>
-          <p>We can help you take your mobile app idea from concept to reality.</p>
+          <img src="./images/performance.svg" alt="performance graph on laptop screen" width="120px" />
+          <h2>Performance Engineering</h2>
+          <p>Are you losing revenue to performance? 53% of mobile site visits are abandoned if a page takes longer than 3 seconds to load. Pinterest reduced load times by 40% and saw a 15% increase in sign ups. Starbucks implemented a 2x faster time to interactive resulting in a 65% increase in rewards registrations.</p>
+          <p><a href="https://developers.google.com/web/fundamentals/performance/why-performance-matters/">Speed matters</a>, but performance engineering and tooling can be hard. Let us help you make your web application faster and implement long-term strategies for maintaining fast web applications. </p>
         </div>
+        <aside>
+          <img src="./images/performance.svg" alt="performance graph on laptop screen" width="79px" />
+          <ul>
+            <li>Performance auditing</li>
```

Comment:
In the last section I say "Usability Audits". Perhaps we should change this to "Performance Audits"?